_rules/active-focus-shortcut-1e9941.md

```diff
@@ -0,0 +1,270 @@
+---
+id: 1e9941
+name: Keyboard shortcut is only active when component has focus
+rule_type: atomic
+
+description: |
+  This rule checks that if keyboard shortcuts using only letter, number, sign or punctuation keys for a user interface component are implemented are only active when that component has focus
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document](https://dom.spec.whatwg.org/#concept-document).
+
+## Expectation
+
+For each target element if [keyboard shortcuts](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-keyboard-shortcut) using only letter, number, sign or punctuation keys for a [user interface component](https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-user-interface-component) are implemented by the target's content are only active when that component has focus

```

Comment:
What is punctuations are written in unicode characters? Perhaps can we define a subset which refers to letter, number, sign and punctuation?